clientconn.go

```diff
@@ -747,6 +747,10 @@ func (cc *ClientConn) handleServiceConfig(js string) error {
 		return err
 	}
 	cc.mu.Lock()
+	if cc.conns == nil {
```

Comment:
Comment here why we need this check, please.  And please explain why we don't check `cc.balancerWrapper` instead?